contrib/windows-cmake/CMakeLists.txt

```diff
@@ -3,48 +3,105 @@ project(hwloc
     LANGUAGES C
     VERSION 2.7.0)
 
-# Available configuration options
-# HWLOC_SKIP_LSTOPO doesn't build/install lstopo
-# HWLOC_SKIP_TOOLS doesn't build/install other hwloc tools
-# HWLOC_SKIP_INCLUDES doesn't install headers
+enable_testing()
 
-set(TOPDIR ../..)
+option(HWLOC_ENABLE_TESTING "Enable testing" ON)
+option(HWLOC_SKIP_LSTOPO "don't build/install lstopo")
```

Comment:
isn't that also a tool? Might consider https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html

Reply:
The Readme calls the variables out distinctly. Maybe intended to be independent options?

Reply:
I wrote that README when extending the vcpkg cmake support a couple weeks ago, and that support didn't build tools or lstopo, but I didn't have any specific reason to split lstopo/tools like this.
Some people may want to only build the lib. Some people may want all tools, including lstopo. Some people may want the lstopo GUI "for fun" and no other command-line tool. I don't see many people wanting other tools but not lstopo (on Unix, lstopo brings Cairo dependencies, but on Windows it doesn't bring anything additional).